pkgs/applications/networking/instant-messengers/matrix-commander/default.nix

```diff
@@ -0,0 +1,36 @@
+{ pkgs, fetchFromGitHub, lib }:
```

Comment:
```suggestion
{ lib, fetchFromGitHub, cacert, python3Packages }:
```

Reply:
I also added stdenv in order to get mkDerivation -- or is there a better way to do that?